cli/src/main.rs

```diff
@@ -246,8 +248,28 @@ fn compile_once(world: &mut SystemWorld, command: &CompileSettings) -> StrResult
     match typst::compile(world) {
         // Export the PDF.
         Ok(document) => {
-            let buffer = typst::export::pdf(&document);
-            fs::write(&command.output, buffer).map_err(|_| "failed to write PDF file")?;
+            if command.output.extension().unwrap_or(OsStr::new("pdf")) == "png" {
+                let pixel_per_pt = 2.0;
+                let pixmaps: Vec<_> = document
+                    .pages
+                    .iter()
+                    .map(|frame| typst::export::render(frame, pixel_per_pt, Color::WHITE))
```

Comment:
Should the PPI and background color be configurable? Transparent background would definitely be useful sometimes.

Reply:
PPI is in the TODO, I didn't think about background color but that makes sense as well.

Reply:
Added to the TODO for discussion